scss/style.scss

```diff
@@ -1,138 +1,147 @@
+$cadetBlue: #429EA6;
+$indigoDye: #153B50;
+$alabaster: #ECEBE4;
+
 body {
     font-family: 'Open Sans', sans-serif;
-    background-color: #f0ecec;
     padding-top: 50px;
 }
 
-.calculator {
-    
+.calculator-container {
     width: 330px;
     height: 430px;
-    background-color: #797979;
+    background-color: $alabaster;
     margin: 0 auto;
-    border-radius: 10px 10px 5px 5px;
-    border: 3px solid rgb(30, 4, 77);
+    padding: 0px 10px;
+    border-radius: 10px 10px 10px 10px;
+    border: 3px solid $indigoDye;
 }
 
-.calculator-form {
+.calculator {
     margin: 0 auto;
-    width: 95%;
-    height: 90%;
-    background-color: #c9c9c9;
-    display: grid;
-    justify-content: center;
-    grid-template-columns: 52px 52px 52px 52px 52px;
-    grid-template-rows: 75px 52px 52px 52px 52px 52px;
-    grid-template-areas: 
-        'output output output output output'
-        'ce ce ce ce ce'
-        'seven eight nine div result'
-        'four five six mul result'
-        'one two three sub result'
-        'zero zero zero sum result';
-    border-radius: 0px 0px 15px 15px;
-    grid-column-gap: 9px;
-    grid-row-gap: 9px;
-    
+    height: 93%;
+    background-color: $cadetBlue;
+    border-radius: 0px 0px 10px 10px;
+
+    &__output {
+        box-sizing: border-box;
+        width: 100%;
+        height: 65px;
+        margin: 0px 0px 9px 0px;
+        color: $alabaster;
+        background-color: $indigoDye;
+        font-size: 24pt;
+        text-align: right;
+    }
+
+    &__action-bar {
+        justify-content: center;
+        display: grid;
+        grid-template-columns: 58px 58px 58px 58px 58px;
+        grid-template-rows: 58px 58px 58px 58px 58px;
+        grid-template-areas: 
+            'ce ce ce ce ce'
+            'seven eight nine div result'
+            'four five six mul result'
+            'one two three sub result'
+            'zero zero zero sum result';
+        grid-column-gap: 6px;
+        grid-row-gap: 7px;
+    }
+}
+
+.calculator-action-bar {
+
     &__item {
-        background-color: #797979;
-        color: rgb(30, 4, 77);
+        background-color: $alabaster;
+        color: $indigoDye;
         border-radius: 6px;
         font-size: 18pt; 
         font-weight: 700;
         cursor: pointer;
-        box-shadow: 0 0 5px 1px rgb(30, 4, 77);
         border:none;
+
         &:hover {
```

Comment:
Did you test hover? Now it looks odd. When I hover any button its text (`+`, `-`, etc.) is painted over `$indigoDye` color.